ccan/ccan/graphql/graphql.h

```diff
@@ -0,0 +1,230 @@
+/* MIT (BSD) license - see LICENSE file for details */
+#ifndef __GRAPHQL_H__
+#define __GRAPHQL_H__ 1
+
+#include <stdio.h>
+
+#include <ccan/list/list.h>
+#include <ccan/tal/tal.h>
+
+// Coding constants
+#define GRAPHQL_SUCCESS 0
```

Comment:
I think defining this as ((const char *)NULL) is a little safer for the callers, so they compare it correctly?

Reply:
Updated to the better ((const char *)NULL)